contracts/token/utils/ERC721_Holder.cairo

```diff
@@ -19,16 +24,22 @@ func onERC721Received{
     return (0x150b7a02)
 end
 
-# ERC721's `safeTransferFrom` requires a means of differentiating between account and
-# non-account contracts. Currently, StarkNet does not support error handling from the
-# contract level; therefore, this ERC721 implementation requires that all contracts that
-# support safe ERC721 transfers (both accounts and non-accounts) include the `is_account` 
-# method. This method should return `0` since this contract is NOT an account.
 @view
-func is_account{
+func supportsInterface{
+        syscall_ptr : felt*,
+        pedersen_ptr : HashBuiltin*,
+        range_check_ptr
+    }(interfaceId: felt) -> (success: felt):
+    let (success) = ERC165_supports_interface('0x150b7a02')
+    return (success)
+end
+
+@constructor
+func constructor{
         syscall_ptr : felt*, 
-        pedersen_ptr : HashBuiltin*, 
+        pedersen_ptr : HashBuiltin*,
         range_check_ptr
-    }() -> (res: felt):
-    return (0)
+    }():
+    ERC165_register_interface('0x150b7a02')
```

Comment:
please comment on what interface this is, and fix the value since it should be a number, not a  string